ui/src/app/components/object-explorer/object-explorer.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
+import { MatTreeModule } from '@angular/material/tree'
+import { MatIconModule } from '@angular/material/icon'
+import { ObjectExplorerComponent } from './object-explorer.component'
+import { ConversionService } from '../../services/conversion/conversion.service'
+import { MatTableModule } from '@angular/material/table'
+import { FormsModule } from '@angular/forms'
+
+xdescribe('ObjectExplorerComponent', () => {
+  let component: ObjectExplorerComponent
+  let fixture: ComponentFixture<ObjectExplorerComponent>
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ObjectExplorerComponent],
+      providers: [ConversionService],
+      imports: [MatTreeModule, MatIconModule, MatTableModule, FormsModule],
+    }).compileComponents()
+  })
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ObjectExplorerComponent)
+    component = fixture.componentInstance
+    component.ngOnChanges({
+      tableNames: {
+        isFirstChange: () => false,
+        currentValue: ['tab1', 'pqr'],
+        previousValue: [],
+        firstChange: false,
+      },
+      conversionRates: {
+        isFirstChange: () => false,
+        currentValue: { tab1: 'GREEN', pqr: 'ORANGE' },
+        previousValue: [],
+        firstChange: false,
+      },
+    })
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
+    expect(component).toBeTruthy()
+  })
+
+  // it('should create true tree node data ', () => {
```

Comment:
please remove this code if not required. If commented for a purpose, please add the purpose here

Reply:
removed